examples/js/exporters/GLTFExporter.js

```diff
@@ -58,19 +58,22 @@ THREE.GLTFExporter.prototype = {
 	constructor: THREE.GLTFExporter,
 
 	/**
-	 * Parse scenes and generate GLTF output
+	 * Parse scenes and generate an object containing the glTF JSON, bin blob, and image urls/file names.
 	 * @param  {THREE.Scene or [THREE.Scenes]} input   THREE.Scene or Array of THREE.Scenes
 	 * @param  {Function} onDone  Callback on completed
+	 * @param  {Function} onError Callback on error
 	 * @param  {Object} options options
 	 */
-	parse: function ( input, onDone, options ) {
+	parseChunks: function ( input, onDone, onError, options ) {
 
 		var DEFAULT_OPTIONS = {
 			binary: false,
-			trs: false,
+			binFileName: "scene.bin",
```

Comment:
Since we're now returning the bin as a blob, and also returning the JSON, this seems trivial to rename:

```
json.buffers[ 0 ].uri = 'foo.bin';
```

...that being the case, is an option for renaming necessary?

Reply:
I think it's a nice option to have. For our editor we need to save the bin file with a unique name so that it doesn't conflict with other files in the export directory. It's nice to not have to read the glTF buffer spec in order to change that file name.